contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -44,52 +44,64 @@ sleep 2
+geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
 
 # unlock solana tester accounts
 address=$(yq -r '.additional_accounts.user_solana.evm_address' config.yml)
 echo "funding solana tester address ${address} with 10000 Ether"
-geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545
+geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
 
 # unlock ethers tester accounts
 address=$(yq -r '.additional_accounts.user_ether.evm_address' config.yml)
 echo "funding ether tester address ${address} with 10000 Ether"
-geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545
+geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
 
 # unlock miscellaneous tests accounts
 address=$(yq -r '.additional_accounts.user_misc.evm_address' config.yml)
 echo "funding misc tester address ${address} with 10000 Ether"
-geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545
+geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
 
 # unlock admin erc20 tests accounts
 address=$(yq -r '.additional_accounts.user_admin.evm_address' config.yml)
 echo "funding admin tester address ${address} with 10000 Ether"
-geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545
+geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
 
 # unlock migration tests accounts
 address=$(yq -r '.additional_accounts.user_migration.evm_address' config.yml)
 echo "funding migration tester address ${address} with 10000 Ether"
-geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545
+geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545 > /dev/null
+
+# unlock local solana relayer accounts
+solana_url=$(yq -r '.rpcs.solana' config.yml)
+solana config set --url "$solana_url" > /dev/null
+
+relayer=$(yq -r '.observer_relayer_accounts.relayer_accounts[0].solana_address' config.yml)
+echo "funding solana relayer address ${relayer} with 100 SOL"
+solana airdrop 100 "$relayer" > /dev/null
+
+relayer=$(yq -r '.observer_relayer_accounts.relayer_accounts[1].solana_address' config.yml)
+echo "funding solana relayer address ${relayer} with 100 SOL"
+solana airdrop 100 "$relayer" > /dev/null
```

Comment:
This throws some errors in the normal non-solana e2e tests:

```
Error: RPC request error: cluster version query failed: error sending request for url (http://solana:8899/): error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution
funding solana relayer address 4kkCV8H38xirwQTkE5kL6FHNtYGHnMQQ7SkCjAxibHFK with 100 SOL
Error: RPC request error: cluster version query failed: error sending request for url (http://solana:8899/): error trying to connect: dns error: failed to lookup address information: Temporary failure in name resolution
```

Consider putting these behind a:
```
if host solana; then
...
fi
```
Like I do in zetaclient: https://github.com/zeta-chain/node/blob/develop/contrib/localnet/scripts/start-zetaclientd.sh#L67

Reply:
I saw this below error when I added the `if` condition. Where (which script) to install the tool for `hotst` cmd? 

`/work/start-zetae2e.sh: line 95: host: command not found`

Reply:
Ah ok add `bind9-host` to `Dockerfile.fastbuild`

Reply:
https://github.com/zeta-chain/node/pull/2673/commits/c9f0b42d134df43a0182fd6d22422973b3d85b42